src/styles/originalHelpdocsStyles.scss

```diff
@@ -1,15 +1,20 @@
 // this is the orginal css provided by HelpDocs for v4 Bars theme. We've edited here, so if you want the actual
 // unedited original css, see the /archiveHelpdocsOriginal folder
 
-* {
+// edited root, it overrides the navbar
+// :root is faster than *, and with it you can target any content, such as SVG, XML, and such.
+:root {
   box-sizing: border-box;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   font-smoothing: always;
   text-rendering: optimizeLegibility;
   -webkit-text-size-adjust: none;
   -ms-text-size-adjust: 100;
-  font-family: -apple-system, BlinkMacSystemFont, "Helvetica Neue", Helvetica, sans-serif
+  font-family: -apple-system,BlinkMacSystemFont, "Montserrat", "Helvetica Neue", Helvetica, sans-serif;
```

Comment:
Just for my knowledge, what is the

-apple-system,BlinkMacSystemFont,

for?

Reply:
Pls refer [here](https://github.com/necolas/normalize.css/issues/665) 
It is default Chrome font on Apple systems used before we load Montesarrat in order to avoid
[font flickering/invisible font](https://web.dev/avoid-invisible-text/) entirely